CONTRIBUTE.md

```diff
@@ -58,14 +58,19 @@ fn div_capped<F: Float>(num: F) {
 
 ## Implement prediction traits
 
-There are two different traits for predictions, `Predict` and `PredictRef`. `PredictRef` takes a reference to the records and produces a new set of targets. This should be implemented by a new algorithms, for example:
+There are three different traits for predictions, `Predict` and `PredictInplace`. `PredictInplace` takes a reference to the records and writes them into targets. This should be implemented by a new algorithms, for example:
```

Comment:
this has to be updates, ~three~ two

Reply:
I would also make the statement more precise:
 `PredictInplace` takes a reference to the records and writes the prediction into a target parameter.

Reply:
Alright. It is better now.